karma.conf.js

```diff
@@ -12,21 +11,26 @@ module.exports = function setKarmaConfig(config) {
     browsers: ['Chrome'],
     colors: true,
     concurrency: Infinity,
+    coverageReporter: {
+      reporters: [
+        { type: 'lcov', dir: '../coverage/', subdir: '.' },
+        { type: 'json', dir: '../coverage/', subdir: '.' },
+        { type: 'text-summary' }
+      ]
+    },
     files: [{
       pattern: testGlob,
       watched: false,
       served: true,
       included: true
-    }],
-    // files: [testGlob, srcGlob],
+    }, testGlob],
     frameworks: ['mocha', 'chai-as-promised', 'chai'],
     logLevel: config.LOG_INFO,
     port: 9876,
     preprocessors: {
-      [testGlob]: ['webpack'] // ,
-      // [srcGlob]: ['webpack']
+      [testGlob]: ['webpack']
     },
-    reporters: ['progress'],
+    reporters: ['mocha', 'coverage'],
```

Comment:
I'm not a huge fan of the `mocha` reporter - I find it too verbose.  What does it display when doing `test:watch`?  If it shows brief output in that case, I'm OK with it, but if it shows verbose output, then I'd prefer not to use it.